src/Robo/Commands/Setup/AllCommand.php

```diff
@@ -25,6 +26,9 @@ public function setup() {
       'source:build',
       'drupal:hash-salt:init',
     ];
+    $commands['drupal:deployment-identifier:init'] = [
+      'id' => RandomString::string(8, FALSE, NULL, 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890!#%^&*()_?/.,+=><'),
```

Comment:
It looks like this command will create a random ID for you if you don't specify one. Why create a random one here?

Reply:
@grasmash that's what we're currently doing as part of the [`drupal:install` command](https://github.com/acquia/blt/blob/ceb6e8e3becf5338a7709d2d528a0fb7128729cc/src/Robo/Commands/Setup/BuildCommand.php#L40), so I just followed suit. Do you know why we were doing that here, previously?

Reply:
Because a Drupal account password is more restrictive it term of the characters that can be used.

Reply:
Hmm, I'm not sure what a Drupal account password has to do with this deployment identifier. Am I missing something?

Reply:
If deployment identifiers don't need to have character restrictions I'll just kill the `id` option altogether. Thoughts?